pennylane/vqe/vqe.py

```diff
@@ -158,20 +158,26 @@ def simplify(self):
         self._ops = ops
 
     def __str__(self):
-        terms = []
+        # lambda function that formats the wires
+        wires_print = lambda ob: ",".join(map(str, ob.wires.tolist()))
+
+        terms_ls = []
 
         for i, obs in enumerate(self.ops):
-            coeff = "({}) [{{}}]".format(self.coeffs[i])
 
             if isinstance(obs, Tensor):
-                obs_strs = ["{}{}".format(OBS_MAP[i.name], i.wires.tolist()[0]) for i in obs.obs]
-                term = " ".join(obs_strs)
+                obs_strs = [
+                    "{}:{}".format(OBS_MAP.get(ob.name, ob.name), wires_print(ob)) for ob in obs.obs
+                ]
+                ob_str = " ".join(obs_strs)
             elif isinstance(obs, Observable):
-                term = "{}{}".format(OBS_MAP[obs.name], obs.wires.tolist()[0])
+                ob_str = "{}:{}".format(OBS_MAP.get(obs.name, obs.name), wires_print(obs))
```

Comment:
Would suggest removing the semicolon to keep printing similar to how [OpenFermion prints QubitOperator objects](https://quantumai.google/reference/python/openfermion/ops/QubitOperator).
```suggestion
                ob_str = "{}{}".format(OBS_MAP.get(obs.name, obs.name), wires_print(obs))
```

Reply:
I added the colon `:` in for two reasons
* When the op is on multiple wires, ex `op0,1`, something just visually seems off
* A name might a number in it.  Same an op is named `Herm0`, then you would get `Herm00`, and that just seems wrong.

Reply:
> When the op is on multiple wires, ex op0,1, something just visually seems off

Could it be added in the multi-qubit case only/only in the Hermitian case?

> A name might a number in it. Same an op is named Herm0, then you would get Herm00, and that just seems wrong.

By default, this would not be the case as the [name of the class object](https://github.com/PennyLaneAI/pennylane/blob/f4f2bd4bec288948118962b6d7227b3d16e88b0b/pennylane/operation.py#L413) is used. The user would have to specifically set the name of the operation to get such values.

Reply:
OK, now removing the color, but using a `'` instead of a `,` between wires, and that seems to look OK.

`Hermitian0'1`